.github/workflows/publish.yml

```diff
@@ -22,6 +22,10 @@ on:
     branches: [ '**' ]
     paths-ignore:
       - 'README.md'
+  merge_group:
```

Comment:
do we want this here? 🤔

Reply:
If this is needed here as well then we should add `merge_group` to [publish.yaml](https://github.com/DataBiosphere/cbas/blob/main/.github/workflows/publish.yml) in CBAS repo as well

Reply:
As discussed in person, just following wherever I see actions being triggered by `on: pull_request`